BentoKit/BentoKit/Screen/ScreenLifecycleEvent.swift

```diff
@@ -0,0 +1,35 @@
+public protocol ScreenLifecycleObserving {
+    func send(_ event: ScreenLifecycleEvent)
+}
+
+public struct ScreenLifecycleEvent {
+    public let status: Status
+    public let isBeingPresentedInitially: Bool
+    public let isBeingRemovedIndefinitely: Bool
```

Comment:
As mentioned in my review of https://github.com/Babylonpartners/ios-playbook/pull/22 IMO we should change `Indefinitely` to `Permanently` for clarity (using a non-negative term).